pkgs/development/libraries/openssl/default.nix

```diff
@@ -43,10 +43,10 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     urls = [
-      "http://www.openssl.org/source/${name}.tar.gz"
-      "http://openssl.linux-mirror.org/source/${name}.tar.gz"
+      "https://www.openssl.org/source/${name}.tar.gz"
+      "https://openssl.linux-mirror.org/source/${name}.tar.gz"
```

Comment:
Note: I think our ``fetchurl`` uses ``--insecure`` parameter, so changing to https is of little use. And since we verify the hash of the result, it wouldn't add any security anyway...